libs/testing/src/ee/clerk-mock-data.ts

```diff
@@ -46,9 +46,10 @@ export const CLERK_USER_1: User = {
   twoFactorEnabled: true,
   locked: false,
   createOrganizationsLimit: 10,
-};
+  deleteSelfEnabled: true,
+} as unknown as User;
```

Comment:
Clerk added some private properties like `_raw` which I can't simulate here and there is also some ECMAScript issue with private types so I just did like this, its only for test mocks anyway.